src/main/java/io/supertokens/webserver/api/totp/VerifyTotpAPI.java

```diff
@@ -61,7 +61,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             super.sendJsonResponse(200, result, resp);
         } catch (LimitReachedException e) {
             result.addProperty("status", "LIMIT_REACHED_ERROR");
-            result.addProperty("retryAfterSec", e.retryInSeconds);
+            result.addProperty("retryAfterMs", e.retryInMs);
```

Comment:
keep the variable name in the exception consistent with the json output. Change it to retryAfterMs

Reply:
Done.